src/containers/Network/BarChartVersion.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  Tooltip,
+  TooltipProps,
+  Label,
+  ResponsiveContainer,
+  Legend,
+  Cell,
+} from 'recharts'
+import {
+  ValueType,
+  NameType,
+} from 'recharts/src/component/DefaultTooltipContent'
+import './css/barchart.scss'
+
+interface Props {
+  data: any[]
+  stableVersion: string | null
+}
+
+interface BarCoordinates {
+  x: number
+  y: number
+}
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+}: TooltipProps<ValueType, NameType>) => {
+  const { t } = useTranslation()
+  if (active) {
+    return (
+      <div className="custom-tooltip">
+        <p className="label">{t('version_display', { version: label })}</p>
+        <p className="value">
+          {t('nodes_count', { count: payload?.[0].payload.count })}
+        </p>
+      </div>
+    )
+  }
+  return null
+}
+
+const renderLegend = (stableVersion: string | null, t: any) => (
+  <div className="legend">
+    <div className="legend-text">
+      <span>{t('current_stable_version')}:</span>
+      <span style={{ color: '#19FF83' }}>
+        {' '}
+        {t('stable_version', { stableVersion })}{' '}
+      </span>
+    </div>
+  </div>
+)
+
+const BarChartVersion = (props: Props) => {
+  const { data, stableVersion } = props
+  const { t } = useTranslation()
+  const [posData, setposData] = useState<BarCoordinates>({ x: 0, y: 0 })
+  const grey = '#9BA2B0'
+  const purple = '#8884d8'
+  const blue = '#1AA4FF'
+  const red = '#FF1A8B'
+  const green = '#19FF83'
```

Comment:
Should these be defined in the css instead of the React code?

Reply:
Some colors will be assigned based on logic and also some field like stroke works differently when inside recharts compared to when set in css so I feel it is cleaner this way. I move those to shared/utils.js where constant are defined also.

Reply:
These should use CSS variables instead. It would probably be cleaner to change the CSS class name instead of defining colors in the code.